controller/controller_server/utils.py

```diff
@@ -428,6 +427,17 @@ def validate_delete_volume_request(request):
     logger.debug("delete volume validation finished")
 
 
+def _validate_node_id(node_id):
+    logger.debug("validating node id")
+
+    split_node = node_id.split(config.PARAMETERS_NODE_ID_DELIMITER)
+
+    if len(split_node) > 4 or len(split_node) < 2:
```

Comment:
```suggestion
    delimiter_count = node_id.count(config.PARAMETERS_NODE_ID_DELIMITER)

    if not 2 <= delimiter_count <= 4:
```

Reply:
`1 <= delimiter_count <= 3` - delimiter is one less than parts..
done